assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -138,4 +138,9 @@ a.inverted {
   .ui.medium.header {
     font-size: 1.45rem;
   }
+}
+
+.image.thumbnail-image {
+  height: 150px;
```

Comment:
why not using the sematic ui size prop ?

Reply:
Because I want to set the height, not the width.